src/components/task-plan/reading.cjsx

```diff
@@ -116,10 +116,20 @@ ReadingPlan = React.createClass
 
   render: ->
     id = @getParams().id
-    footer = <BS.Button bsStyle="primary" onClick={@publish}>Publish</BS.Button>
+
+    if id
+      plan = TaskPlanStore.get(id)    
+    else
+      plan = TaskPlanStore.get(@state.id)
+     
+    if plan.title and plan.due_at and plan.settings.page_ids.length > 0
+    then btnDisable = '' else btnDisable = 'disabled'
```

Comment:
FYI: There is already an existing syntax in tutor-js for optionally adding classes: https://github.com/openstax/tutor-js/blob/master/src/components/question.cjsx#L36